test/httpRequestTrigger.paging.spec.js

```diff
@@ -225,6 +225,58 @@ describe('httpRequest action paging', () => {
     expect(messagesNewMessageWithBodyStub.args[0][0]).to.be.eql(responseMessage);
   });
 
+  it('paging loop, 3 times, POST request', async () => {
+    const messagesNewMessageWithBodyStub = stub(
+      messages,
+      'newMessage',
+    ).returns(Promise.resolve());
+    const msg = {
+      id: '123456789',
+      data: {
+        url: 'http://example.com',
+      },
+    };
+    const cfg = {
```

Comment:
Let's use the raw body config in this as well to test the scenario where we are sending the page in the payload

Reply:
Not sure how I missed passing in the body on a POST request